### PR TITLE
added a remote alias

### PR DESCRIPTION
**Pull request by FarhadG**
@AndreMilota can you please review this code and merge into main if appropriate.

### CODE REVIEW

**Comment by AndreMilota** [comment]
looks good what happened on the first line?

File: alias.sh [hunk 1]
```diff
@@ -50,4 +50,5 @@ alias gpom:h="git push origin main; git push heroku main"
 alias gpom:gh="git push origin main; git push origin main:gh-pages -f"
 alias gsur="git submodule update --init --recursive"
 alias ghist="git log --graph --abbrev-commit --decorate --date=relative --format=format:'%C(bold blue)%h%C(reset) - %C(bold green)(%ar)%C(reset) %C(white)%s%C(reset) %C(dim white)- %an%C(reset)%C(bold yellow)%d%C(reset)' --all"
-alias grst="git reset HEAD ."
\ No newline at end of file
+alias grst="git reset HEAD ."
```
Comment (AndreMilota) [inline alias.sh:53]: so you added a return

**Comment by FarhadG** [comment]
> looks good what happened on the first line?

Thank you! I'll look at it in a bit

**Review by AndreMilota — CHANGES_REQUESTED** [review]

File: alias.sh (hunk 1 above)
Comment (FarhadG) [reply alias.sh:53]: You got it! It's the result of a `/n` at that line.

**Pull request merged**